Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.FileSystemChange.cs

```diff
@@ -13,7 +14,7 @@ internal abstract partial class CommonProjectNode
         /// Represents an individual change to the file system.  We process these in bulk on the
         /// UI thread.
         /// </summary>
-        [DebuggerDisplay("{nameof(FileSystemChange)}: {isRename} {Type} {path}")]
+        [DebuggerDisplay("FileSystemChange: {isRename} {Type} {path}")]
```

Comment:
>FileSystemChange [](start = 26, length = 16)

Why not `nameof`?

Reply:
doesn't appear to work in string literals.

Reply:
It's probably `DebuggerDisplayAttribute` that doesn't support it.